WIP Neworigins v3 changes

artyomtrityak opened the pull request:
This is changelog for New Origins V3

1. Increase TURTLE walking capacity to be same as swimming ie 115
2. Reduce LBOW bonus by 1 from +0 to -1
3. Slightly increase undead after raised after combat
4. Enable level scaling for Quartermasters: increase range and decrease cost 
5. Increase low-end mosters spoils from 50 to 60 so some normal type items can drop from them
6. Increase TELEPORT 30 -> 50 weight per level
7. Enable weather
8. Make Phantom lich and phantom dragons to be max = skill level instead of 1.

atrytiak committed:
Change TURT walking capacity to be same as swimming 115

atrytiak committed:
Reduce LBOW bonus by 1 from 0 to -1

atrytiak committed:
Changes to ruleset

atrytiak committed:
Increase spoils

atrytiak committed:
Increase fish baseprice

atrytiak committed:
Add weather, increase illusory max and teleport

atrytiak committed:
Fix phantom lich description

atrytiak committed:
Reduce WING amount

artyomtrityak commented:
Can you please change it towards  neworigins-v3 and wait for Overwhelming to be merged

valdisz committed:
Free moving item (#94)

* free moving items

* adjusted CATP and STED definition

* adjusted FMI

* FMI description and spoils

valdisz commented on gamedata.cpp:
```diff
@@ -310,7 +310,7 @@ ItemType id[] =
 	 "FISH",1,1,1, {{-1,0}, {-1,0},{-1,0},{-1,0}},
 	 NULL,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 1, IT_NORMAL | IT_FOOD, 15,0,
+	 1, IT_NORMAL | IT_FOOD, 20,0,
```
IMO it is better to change such values in the ruleset folder via `Modify*` methods. Then there will be no need to revert changes if we will merge changes back to the upstream Atlantis repo.

valdisz committed:
Overwhelming (#92)

valdisz committed:
fixed arcadia so that it can be built (#97)

atrytiak committed:
Add gate changes and magic armor

valdisz committed:
Multi hit damage and weapon BM (#99)

* Hit damage modifier (#93)

* attack damage can be more than 1

* added `hitDamage` to all relevant tables and using it in battle

* weapon bonus/malus

* hit damage description

* weapon BM table modification method

* weapon BM report

* fixes after rebase

* PR comments

* not STD :(

* BM example

* using AString to compare strings

* added GetWeaponBonusMalus to the header

* added std back

* use int instead of int32_t

* fixed warnings and incorrect header file

* added fogottend damage anount for the effect which caused segfault

atrytiak committed:
GATE jump improvements

atrytiak committed:
Merge branch 'neworigins-v3' of github.com:artyomtrityak/Atlantis into neworigins-v3

valdisz committed:
fixed FMI battle (#100)

atrytiak committed:
Add attack from behind if overwhelmed

atrytiak committed:
HEAL ALL

atrytiak committed:
OH4

atrytiak committed:
Strict guard

atrytiak committed:
Disable ocean guard

valdisz committed:
Detailed battle statistics (#101)

* battle stats

* weapon stats

* battle stats

* battle stats settings

atrytiak committed:
Add multi damage weapons

atrytiak committed:
weapons redo

atrytiak committed:
No ENDU?

valdisz committed:
fixes segfault when special attack weapon is used (#104)

atrytiak committed:
reduce combat bonus and add variable attacks

atrytiak committed:
adjust armorpiercing

atrytiak committed:
regular sword increase attacks number

atrytiak committed:
change armor protection

atrytiak committed:
fix game data for v3

atrytiak committed:
Adjust v3 terrain

atrytiak committed:
Increase health summoned monsters to compensate multi attacks melee

atrytiak committed:
increase comb summoned monsters

valdisz committed:
fixed segfault (#105)

atrytiak committed:
monsters boost

atrytiak committed:
Merge branch 'neworigins-v3' of github.com:artyomtrityak/Atlantis into neworigins-v3

atrytiak committed:
Update monsters

atrytiak committed:
Add extended fortifications with some logs

atrytiak committed:
Modify buildings

atrytiak committed:
ADMT dragon and stelath

atrytiak committed:
Updated v3: portal and speed

atrytiak committed:
Remove trade items from spoils

atrytiak committed:
remove UW

atrytiak committed:
Increase ship capacity

atrytiak committed:
make mages and their guard aggressive

atrytiak committed:
Increase dangerous

atrytiak committed:
devil

atrytiak committed:
Fix catapults

atrytiak committed:
Monsters recovery and armor

atrytiak committed:
increase chance for flaming sword / runesword to be cast

Logintmg committed:
Monster and terrain adjustments (#107)

livestock - removed no_transport
turtle - loot value decreased (they drop more often)
power of monsters that drop advanced or magic items increased, their loot value increased too
tundra wild monster spawn % increased, lair amount reduced and giantcastle added
volcano terrain adjusted spawns, resources etc.
wing ridi bonus reduced to +3 max

atrytiak committed:
post merge fixes

atrytiak committed:
Living water ranged protection

atrytiak committed:
Remove RGEMS and stabilize market

atrytiak committed:
Update local transport

atrytiak committed:
fix ships and local

atrytiak committed:
Change TURT

valdisz committed:
Parametric map generator (#106)

* parametric map generator

* fallback implementation for other rulesets

* changed limit constraints

* fixed typo

* changes

* volcanoes

* lakes

* do not make huge straits

* volcanoes

* dynamic strait size and limits

* typo

* no UW

* for temp map report

* more varaety and better biome selection

* cleanup

* island core distance

valdisz committed:
more parameters (#108)

Logintmg committed:
monster adjustments (#111)

1.hydra tact3->tact4
2.ice dragon icebreath4->6  tact4->tact5
3.cloud giant lightning3->2 comb 6->7 tact 4->5
4.some obse changes

valdisz committed:
init struct (#113)

atrytiak committed:
Fix mountains and volcano

valdisz committed:
missing return (#114)

atrytiak committed:
Merge branch 'neworigins-v3' of github.com:artyomtrityak/Atlantis into neworigins-v3

atrytiak committed:
Magic and mountain
